src/nvme/mi.h

```diff
@@ -367,6 +367,34 @@ struct nvme_mi_admin_resp_hdr {
 	__le32	cdw0, cdw1, cdw3;
 } __attribute__((packed));
 
+/**
+ * struct nvme_mi_control_req - The Control Primitive request.
+ * @hdr: Generic MI message header
+ * @opcode: Control Primitive Opcodes (using &enum nvme_control_opcode)
+ * @tag: flag - Be used to sent and return, reserved now
```

Comment:
Not sure this description hre (and below) makes sense. How about: 

```@tag: opaque value passed from request to response```

Reply:
done